clang/test/SemaSYCL/intel-fpga-local.cpp

```diff
@@ -567,70 +309,13 @@ void diagnostics()
   //expected-error@+1{{merge direction must be 'depth' or 'width'}}
   [[intel::merge("mrg4", "depths")]] unsigned int mrg_invalid_arg[4];
 
-  //Last one is applied and others ignored.
-  //CHECK: VarDecl{{.*}}mrg_mrg
-  //CHECK: SYCLIntelMergeAttr{{.*}}"mrg4" "depth"{{$}}
-  //CHECK: SYCLIntelMergeAttr{{.*}}"mrg5" "width"{{$}}
-  //expected-warning@+2{{attribute 'merge' is already applied}}
-  [[intel::merge("mrg4", "depth")]]
-  [[intel::merge("mrg5", "width")]] unsigned int mrg_mrg[4];
-
   // bank_bits
-  //CHECK: VarDecl{{.*}}bankbits
-  //CHECK: SYCLIntelNumBanksAttr{{.*}}Implicit{{$}}
-  //CHECK-NEXT: IntegerLiteral{{.*}}16{{$}}
-  //CHECK: SYCLIntelMemoryAttr{{.*}}Implicit
-  //CHECK: SYCLIntelBankBitsAttr
-  //CHECK-NEXT: ConstantExpr
-  //CHECK-NEXT: value:{{.*}}2
-  //CHECK-NEXT: IntegerLiteral{{.*}}2{{$}}
-  //CHECK-NEXT: ConstantExpr
-  //CHECK-NEXT: value:{{.*}}3
-  //CHECK-NEXT: IntegerLiteral{{.*}}3{{$}}
-  //CHECK-NEXT: ConstantExpr
-  //CHECK-NEXT: value:{{.*}}4
-  //CHECK-NEXT: IntegerLiteral{{.*}}4{{$}}
-  //CHECK-NEXT: ConstantExpr
-  //CHECK-NEXT: value:{{.*}}5
-  //CHECK-NEXT: IntegerLiteral{{.*}}5{{$}}
-  [[intel::bank_bits(2, 3, 4, 5)]] unsigned int bankbits[64];
-
-  //CHECK: VarDecl{{.*}} bb_reg 'unsigned int[4]'
-  //CHECK: SYCLIntelNumBanksAttr{{.*}}Implicit
-  //CHECK-NEXT: IntegerLiteral{{.*}}'int' 4
-  //CHECK: SYCLIntelMemoryAttr{{.*}}Implicit Default
-  //CHECK: SYCLIntelBankBitsAttr
-  //CHECK-NEXT: ConstantExpr{{.*}}'int'
-  //CHECK-NEXT: value: Int 2
-  //CHECK-NEXT: IntegerLiteral{{.*}}'int' 2
-  //CHECK-NEXT: ConstantExpr{{.*}}'int'
-  //CHECK-NEXT: value: Int 3
-  //CHECK-NEXT: IntegerLiteral{{.*}}'int' 3
   //expected-error@+2 1{{'fpga_register' and 'bank_bits' attributes are not compatible}}
   [[intel::bank_bits(2, 3)]]
   [[intel::fpga_register]]
   //expected-note@-2 1{{conflicting attribute is here}}
   unsigned int bb_reg[4];
 
-  //CHECK: VarDecl{{.*}}bb_bb
-  //CHECK: SYCLIntelBankBitsAttr
-  //CHECK-NEXT: ConstantExpr
-  //CHECK-NEXT: value:{{.*}}42
-  //CHECK-NEXT: IntegerLiteral{{.*}}42{{$}}
-  //CHECK-NEXT: ConstantExpr
-  //CHECK-NEXT: value:{{.*}}43
-  //CHECK-NEXT: IntegerLiteral{{.*}}43{{$}}
-  //CHECK: SYCLIntelBankBitsAttr
-  //CHECK-NEXT: ConstantExpr
-  //CHECK-NEXT: value:{{.*}}1
-  //CHECK-NEXT: IntegerLiteral{{.*}}1{{$}}
-  //CHECK-NEXT: ConstantExpr
-  //CHECK-NEXT: value:{{.*}}2
-  //CHECK-NEXT: IntegerLiteral{{.*}}2{{$}}
-  //expected-warning@+2{{attribute 'bank_bits' is already applied}}
```

Comment:
Same here

Reply:
Yes, AST part we are checking inside clang/test/SemaSYCL/intel-fpga-local-ast.cpp file at line 240-257
Diagnostic part we are checking inside clang/test/SemaSYCL/intel-fpga-local.cpp file at line 116-119